src/modules/75-cf/components/ListingPageTemplate/ListingPageTemplate.tsx

```diff
@@ -6,20 +6,23 @@
  */
 
 import React, { ReactNode, useMemo } from 'react'
-import { Breadcrumb, Page } from '@harness/uicore'
+import { Breadcrumb, Layout, Page, Text } from '@harness/uicore'
+import { Color, FontVariation } from '@harness/design-system'
 import { getErrorMessage } from '@cf/utils/CFUtils'
+import graduationCapIcon from '@cf/images/icons/graduation_cap.svg'
 import { NGBreadcrumbs } from '@common/components/NGBreadcrumbs/NGBreadcrumbs'
 import { ContainerSpinner } from '@common/components/ContainerSpinner/ContainerSpinner'
 import { useDocumentTitle } from '@common/hooks/useDocumentTitle'
+import { useStrings } from 'framework/strings'
 import ListingPageHeading from './ListingPageHeading'
-
 import css from './ListingPageTemplate.module.scss'
 
 export interface ListingPageTemplateProps {
   breadcrumbs?: Breadcrumb[]
   title: string
   titleTooltipId?: string
   headerContent?: ReactNode
+  headerActions?: { docs?: string; video?: string }
```

Comment:
Might be a good idea to make sure the name of these reflects their use
```suggestion
  headerActions?: { docsURL?: string }
```

Reply:
I'm still figuring out how this should look - docs don't currently need to pass a custom label text but the video action buttons will each have their own label text